### PR TITLE
Add support for media:thumbnail as episode_art_url.

### PR DESCRIPTION
**Pull request by auouymous**

### CODE REVIEW

File: podcastparser.py [hunk 1]
```diff
@@ -179,6 +179,14 @@ def start(self, handler, attrs):
             handler.set_episode_attr(self.key, self.filter_func(value))
 
 
+class EpisodeAttrFromUrl(Target):
+    def start(self, handler, attrs):
+        value = attrs.get('url')
+        if value:
+            value = urlparse.urljoin(handler.base, value)
+            handler.set_episode_attr(self.key, self.filter_func(value))
```
Comment (thp) [inline podcastparser.py:187]: Maybe do it like this to share code:

```
class EpisodeAttrFromHref(Target):
    ATTRIBUTE = 'href'

    def start(self, handler, attrs):
        value = attrs.get(self.ATTRIBUTE)
        if value:
            value = urlparse.urljoin(handler.base, value)
             handler.set_episode_attr(self.key, self.filter_func(value))


class EpisodeAttrFromUrl(EpisodeAttrFromHref):
    ATTRIBUTE = 'url'
```

**Review by thp — CHANGES_REQUESTED** [review]
One comment to avoid code duplication. Rest looks good.

File: podcastparser.py (hunk 1 above)
Comment (auouymous) [reply podcastparser.py]: Good idea, done.

**Comment by elelay** [comment]
Better with a test ;-)

**Comment by auouymous** [comment]
Added some tests.

**Comment by elelay** [comment]
Merged, thanks :-)

**Pull request merged**